.github/workflows/unit_tests.yaml

```diff
@@ -14,8 +14,9 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
-          - 2.5
```

Comment:
Dropping something is usually important to mention. From experience I can tell you that if you read the changelog and only see "Add X" but then later figure out it included "Drop Y" while you relied on Y that it's a nasty surprise.

Because of that I tend to separate the changes.

In addition to that I think it's a good practice to include `required_ruby_version` in the gemspec to match what it tests against.

Reply:
Yes, make sense. Here is it #47. Also, i'll do this in other hammer-cli PRs too.